src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -1,6 +1,11 @@
 # This file defines the default (English) variants of all of the internationalised messages
 pscanalpha.desc = Passive Scan Rules - alpha
 
+pscanalpha.blanktarget.name=Link uses _blank target
+pscanalpha.blanktarget.desc=At least one link on this page uses a target of "_blank" without using rel="noopener" which allows the target page to take control of this page.
```

Comment:
`without using keyword "noopener" in "rel" attribute`? (or something like that)

Reply:
Ya that probably makes more sense. Then the user isn't left wondering if the quotes demark what's important and also doesn't limit the values of "rel".